src/System.Configuration.ConfigurationManager/src/System/Configuration/IConfigurationSystem.cs

```diff
@@ -6,8 +6,6 @@
 
 namespace System.Configuration
 {
-    // obsolete
-    [ComVisible(false)]
```

Comment:
ComVisible is used for interop, not security. Are these intended to be in this change?